tools/make-deb.sh

```diff
@@ -15,18 +15,29 @@ BUILD="$(mktemp -d)"
 mkdir -p "${BUILD}/opt"
 cp -a /opt/boulder "${BUILD}/opt/boulder"
 
+# Determine architecture - use ARCH env var if set, otherwise detect from uname
+if [ -n "${ARCH:-}" ]; then
+    DEB_ARCH="${ARCH}"
+else
+    case "$(uname -m)" in
+        "x86_64") DEB_ARCH="amd64" ;;
+        "aarch64"|"arm64") DEB_ARCH="arm64" ;;
+        *) echo "Unsupported architecture: $(uname -m)" && exit 1 ;;
+    esac
+fi
+
```

Comment:
make-deb.sh is a transitional tool to keep building .debs until we transition to running containers.

I think it's okay to assume `$ARCH` will always be set, because the only place we call it from is `container-build.sh`, which sets it.